cmd/testsuite/cmd/test.go

```diff
@@ -164,22 +159,22 @@ func writeJUnitReport(junitPath string, testSuites *junit.Testsuites) error {
 	return nil
 }
 
-func writeBenchmarkReport(benchmarkPath string, report *eventbenchmark.GlobalBenchmarkReport) error {
-	benchmarkFile, err := os.Create(benchmarkPath)
-	if err != nil {
-		return errors.WithStack(err)
-	}
-	defer benchmarkFile.Close()
-
-	data, err := report.Generate(eventbenchmark.YamlFormatter)
-	if err != nil {
-		return errors.WithStack(err)
-	}
-
-	_, err = benchmarkFile.Write(data)
-	if err != nil {
-		return errors.WithStack(err)
-	}
-
-	return nil
-}
+// func writeBenchmarkReport(benchmarkPath string, report *eventbenchmark.GlobalBenchmarkReport) error {
```

Comment:
do we need to keep this in the source?

Reply:
We don't. But I'm hoping to re-enable it shortly.